src/api/middlewares/validators/validateUser.js

```diff
@@ -0,0 +1,18 @@
+import { z } from "zod";
+
+// User 유효성 검사 미들웨어는 zod 활용해서 구현해봄
+// 이후에 시간 남으면 다른 유효성 검사도 zod로 바꾸기
+export const signupSchema = z.object({
+  email: z.email({ message: "유효한 이메일 형식이 아닙니다." }),
```

Comment:
zod로 유효성 검증하는 것은 좋습니다!! 다만 zod에서 email을 인증하는데 제안과 같이 작성해야하지 않을까요?
```suggestion
  email: z.string().email({ message: "유효한 이메일 형식이 아닙니다." }),
```